packages/@aws-cdk/aws-apigatewayv2/lib/common/integration.ts

```diff
@@ -9,4 +9,36 @@ export interface IIntegration extends IResource {
    * @attribute
    */
   readonly integrationId: string;
-}
\ No newline at end of file
+}
+
+/**
+ * AWS service integration sub types
+ * https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services-reference.html
+ */
+export enum AwsServiceIntegrationSubtype {
```

Comment:
As mentioned in another comment above, we should not define this type and be specific for each integration.

Reply:
See below re: renderRequestParameters, and if it still makes sense to remove.